css/style.css

```diff
@@ -0,0 +1,511 @@
+  display: inline;
+  align-items: center;
+  background: #ebebff;
+  border-radius: 8px;
+  width: 49px;
+  height: 20px;
+  font-size: 15px;
+  padding: 4px 12px;
+  color: #6070ff;
+}
+
+.btn-one {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 4%;
+  font-size: 17px;
+  font-weight: 300;
+  text-transform: capitalize;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+.btn-two {
+  border: 1px solid #4053fc;
+  padding: 7px;
+  border-radius: 4%;
+  font-size: 17px;
+  font-weight: 300;
+  text-transform: capitalize;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+a.btn-one {
+  text-decoration: none;
+  color: #4053fc;
+}
+
+a.btn-two {
+  text-decoration: none;
+}
+
+.btn-tags {
+  margin-bottom: 30px;
+}
+
+.section-about {
+  background-color: white;
+  border-top-right-radius: 100px;
+}
+
+.section-about h1 {
+  position: static;
+  width: 920px;
+  height: 52px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 35px;
+  letter-spacing: 0.37px;
+  color: #091e42;
+  margin: 40px 10px 30px 0;
+  font-size: 35px;
+}
+
+.about-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(420px, 1fr));
+  grid-gap: 20px;
+  align-items: center;
+}
+
+.about-grid > article {
+  padding-bottom: 50px;
+}
+
+.contact-form {
+  background-color: #fff;
+}
+
+.contacts-form {
+  background: url("../image/contactbackground.png");
+  background-size: 60% 100%;
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  border-top-left-radius: 100px;
+  width: 100%;
+  padding: 13px 24px 0;
+  box-sizing: border-box;
+}
+
+.contacts-form h1:first-child {
+  text-align: center;
+  font-weight: bold;
+  font-size: 2.5rem;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+.contacts-form h1:first-of-type {
+  color: #ebebff;
+  font-size: 2.5rem;
+  text-align: center;
+  font-weight: bold;
+  margin-top: 80px;
+}
+
+.contacts-form p:nth-child(2) {
+  color: #fff;
+  text-align: center;
+  font-size: 1.25rem;
+  margin-bottom: 40px;
+}
+
+.email {
+  width: 100%;
+  height: 48px;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: none;
+  padding: 15px;
+  font-size: 1.25rem;
+  color: #172b4d;
+  box-sizing: border-box;
+}
+
+.msg {
+  width: 100%;
+  padding: 16px;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin-bottom: 22px;
+  color: black;
+  font-family: Poppins, sans-serif;
+  font-size: 1.25rem;
+}
+
+.email:focus,
+.message:focus {
+  border: none;
+  outline: none;
+}
+
+.name {
+  width: 100%;
+  height: 48px;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: none;
+  padding: 15px;
+  font-size: 17px;
+  color: #172b4d;
+  box-sizing: border-box;
+}
+
+.send {
+  color: #6070ff;
+  font-size: 1.06rem;
+  border-radius: 8px;
+  margin-bottom: 32px;
+  border: none;
+  padding: 14px;
+}
+
+.send:hover {
+  border: 1px solid white;
+}
+
+@media only screen and (max-width: 769px) {
```

Comment:
- [ ] I kindly remind you, consider the mobile phone when you work on the desktop size, this should work as usual when you finish your desktop size, following your own deployment, this is how the mobile size looks using chrome web tools. I kindly invite you to work on this. 🐦 
![image](https://user-images.githubusercontent.com/20563252/126811072-c06935ee-f5d1-46bd-8fb2-31d02f65959d.png)